src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -311,6 +312,14 @@ namespace HVACUnitaryBypassVAV {
         } else {
             CalcCBVAV(CBVAVNum, FirstHVACIteration, PartLoadFrac, QSensUnitOut, OnOffAirFlowRatio, HXUnitOn);
         }
+        if (CBVAV(CBVAVNum).modeChanged) {
+            // set outlet node SP for mixed air SP manager
+            DataLoopNode::Node(CBVAV(CBVAVNum).AirOutNode).TempSetPoint = CalcSetPointTempTarget(CBVAVNum);
+            if (CBVAV(CBVAVNum).OutNodeSPMIndex > 0) { // update mixed air SPM if exists
+                SetPointManager::MixedAirSetPtMgr(CBVAV(CBVAVNum).OutNodeSPMIndex).calculate(); // update mixed air SP based on new mode
+                SetPointManager::UpdateMixedAirSetPoints(); // need to know control node to fire off just one of these, do this later
+            }
+        }
```

Comment:
Note the modeChanged flag, that will be apparent later. Here the outlet node SP is set and the specific SPM is called. I didn't go to the length needed to Update the specific SPM and just update them all.